ci/pipeline/setup

```diff
@@ -3,3 +3,12 @@ chmod +x gradlew
 echo "org.gradle.caching=true" >>gradle.properties
 echo "org.gradle.jvmargs=-Xmx7g"
 touch local.properties
+touch build-logic/signing_info.gradle.kts
+# Add dummy signing info to build-logic/signing_info.gradle.kts
+cat << EOF >> build-logic/signing_info.gradle.kts
+extra.apply {
+    set("store_file", "dummy")
```

Comment:
Wouldn't this break debug key signing?

Reply:
No it won't break anything. I added the dummey data for the unit test and sonarqube steps to pass.
The real signing data is being added by the `ci/deployment/signing_properties_setup` script